recipes/nyuki_recipe/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "nyuki" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source: 
+  url: https://github.com/00krishna-tools/nyuki/archive/v{{ version }}.tar.gz
+  sha256: 88e943eed59d5ca95d6716b22b47c71a7b7c9323a71348a4623e329bb66d7ae7
+
+build:
+  noarch: python
+  number: 0
+#  skip: true  # [py<36]
```

Comment:
```suggestion
```

Reply:
Yep, I made this change. Thanks @chrisburr . The linter is happy now too :).